### DISCUSSION TITLE
Popup panel open in narrow mode on Nightly lately

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2385] [General] [answered]
Does it happen in other browsers for you (uBO bug), or is it something with Nightly?

![image](https://user-images.githubusercontent.com/886325/203770966-d28ee39e-0658-404c-b76f-5ad6508ea61a.png)

**Comment by uBlock-user** [comment]
Happened to me few years back. No way to reproduce. When it happens the popup gets cut off.

**Comment by gorhill** [comment]
It has happened to me too lately.

**Comment by krystian3w** [comment]
In my case, long button descriptions sometimes extend the panel by 3-4 times the standard width.

It also sometimes happens that the height is higher than the number of elements in the pop-up by 1 row of 'show more/less' buttons.

**Comment by ghost** [comment, marked as answer]
This seems to work: https://github.com/gorhill/uBlock/commit/92397baa3901f2a18564894098628955b2661e78

**Reply by gorhill** [reply]
Sorry, forgot to mention discussion in commit message.

**Reply by krystian3w** [reply]
Seems work.
